### ISSUE TITLE
Migrate to Vue 3.0

### ISSUE THREAD

**Issue by TanayB11** [TanayB11/jadeocr#57]
Vue 3.0 is in the Release Candidate stage, and we should switch to it. Here's why:

1. Faster compiler: Better development experience for new features. It checks for capitalization instead of going down the tree to check if a tag is component.
2. Better tree-shaking: Gets rid of unused bloat code
3. Better TypeScript support: TS is good, we should switch to it. The compiler itself is written in TS.
4. Better native support: We can bring jadeocr to mobile!

@kongriley what are your thoughts?

`:wq`

**Comment by TanayB11** [comment]
* We should switch _after_ it releases, and after we finish the CAC entry.

**Comment by kongriley** [comment]
I agree, and we should switch as soon as possible (after release/finishing CAC of course) for better performance.
I'm particularly excited about jadeocr on mobile - it will allow users to easily draw on the go / using an iPad, which is better for handwriting.
